transcript/src/basic.rs

```diff
@@ -32,15 +32,15 @@ impl<E: ExtensionField> Transcript<E> for BasicTranscript<E> {
     }
 
     fn read_challenge(&mut self) -> Challenge<E> {
-        // notice `from_bases` and `from_limbs` has the same behavior but
```

Comment:
The rewording here is all in https://github.com/scroll-tech/ceno/pull/770 and doesn't have much to do with your PR.  (It'll go away in the diff, once we merge https://github.com/scroll-tech/ceno/pull/770 into master.)